go.mod

```diff
@@ -33,10 +33,9 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/instrumenta/kubeval v0.0.0-20190918223246-8d013ec9fc56
-	github.com/jackc/pgconn v1.13.0
 	github.com/jackc/pgerrcode v0.0.0-20201024163028-a0d42d470451
 	github.com/jackc/pgtype v1.13.0
-	github.com/jackc/pgx/v4 v4.17.2
+	github.com/jackc/pgx/v4 v4.17.2 // indirect
```

Comment:
do we know what's bringing in a dependency on v4?

Reply:
I took a quick look at `go mod graph`, and it looks like `github.com/jackc/pgtype` is pulling it in - even in the latest tagged version (1.14) - see  https://github.com/jackc/pgtype/blob/master/go.mod

Reply:
I assume that pgx/v5 works fine with the old pgxtype?  If so I'd leave it for now and see if we can migrate to pgx/v5/pgtype in a separate pr.